frameworklauncher/src/test/resources/TestExpects/LauncherConfiguration.yml

```diff
@@ -2,30 +2,17 @@
 amAllowNoneGpuJobOnGpuNode: false
 amAttemptFailuresValidityIntervalSec: 0
 amAttemptMaxCount: 0
+amCandidateNodesFactor: 0
```

Comment:
>0 [](start = 24, length = 1)

0  is the default value ? default 0 will cause none candidate node selected.

Reply:
It is just UT yaml

---
In reply to: [193963658](https://github.com/Microsoft/pai/pull/685#discussion_r193963658) [](ancestors = 193963658)